storage/client_test.go

```diff
@@ -215,6 +215,19 @@ func incrementArgs(key []byte, inc int64, raftID int64, storeID proto.StoreID) (
 	return args, reply
 }
 
+func internalTruncateLogArgs(index uint64, raftID int64, storeID proto.StoreID) (
```

Comment:
Not without exporting the functions. `storage/client_*_test.go` is in package `storage_test` but the rest of the tests are in package `storage`. It might be worth moving these into a sub-package (and exporting them) so they can be shared. We could make the new package internal so it couldn't be used from anywhere else.